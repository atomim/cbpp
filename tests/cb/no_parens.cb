#define MAKRO(a) a*a
AddText "this is a simple line " + (MAKRO(5)) 
DrawScreen
WaitKey
