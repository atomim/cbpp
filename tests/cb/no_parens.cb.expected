// #define MAKRO(a) a*a
AddText "this is a simple line " + (5*5) 
DrawScreen
WaitKey
